<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="win">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="w_tt" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CSideBox" name="w_lt" native="true"/>
       </item>
       <item>
        <widget class="CSideBox" name="w_t" native="true"/>
       </item>
       <item>
        <widget class="CSideBox" name="w_rt" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="content">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="CSideBox" name="w_l" native="true"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="title">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>614</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="min">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/frame/min.png</normaloff>:/frame/min.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="rest">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/frame/rest.png</normaloff>:/frame/rest.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="max">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/frame/max.png</normaloff>:/frame/max.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="close">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/frame/close.png</normaloff>:/frame/close.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Form" name="form"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CSideBox" name="w_r" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="w_bb" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CSideBox" name="w_lb" native="true"/>
       </item>
       <item>
        <widget class="CSideBox" name="w_b" native="true"/>
       </item>
       <item>
        <widget class="CSideBox" name="w_rb" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSideBox</class>
   <extends>QWidget</extends>
   <header>csidebox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Form</class>
   <extends>QFrame</extends>
   <header>form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
